app/Helpers/TerrafundDashboardQueryHelper.php

```diff
@@ -2,11 +2,17 @@
 
 namespace App\Helpers;
 
+use App\Models\V2\Projects\Project;
+
 class TerrafundDashboardQueryHelper
 {
-    public static function buildQueryFromRequest($query, $request)
+    public static function buildQueryFromRequest($request)
     {
+        $query = Project::query();
         $query = $query->where('framework_key', 'terrafund');
+        $query = $query->whereHas('organisation', function ($query) {
+            $query->whereIn('type', ['for-profit-organization', 'non-profit-organization']);
+        });
         if ($request->has('country')) {
             $country = $request->input('country');
             $query->where('country', $country);
```

Comment:
I see that this is older code, but is this functioning correctly? I had thought $query was immutable and you would need to save the return value. Maybe I'm misunderstanding how the query builder works, but I would think this is necessary:
```
$query = $query->where('country', $country);
```
(and same change in the `elseif` block)